src/passes/Inlining.cpp

```diff
@@ -1079,20 +1058,29 @@ struct Inlining : public Pass {
     });
   }
 
-  bool worthInlining(Name name) {
+  InliningMode getInliningMode(Name name) {
+    auto& info = infos[name];
+
+    if (info.inliningMode != InliningMode::Unknown) {
```

Comment:
Where/how is the default value set to Unknown? I'd guess we need line 69 to change from
```cpp
InliningMode inliningMode;
```
to
```cpp
InliningMode inliningMode = InliningMode::Unknown;
```
But perhaps there is some C++ default value spec thing I'm forgetting?

Reply:
The FunctionInfo constructor calls `clear()` where are all the initializations are. So I had the the initialization there next to other field initializations.

Reply:
Right, sorry, I forgot we have that pattern here... iirc it's because the atomic one can't be initialized inline so for consistency we moved them all to stay together.